lib/open_food_network/order_cycle_permissions.rb

```diff
@@ -196,12 +262,14 @@ def editable_variants_for_outgoing_exchanges_to(hub)
     private
 
     def user_manages_coordinator_or(enterprise)
-      managed_enterprises.pluck(:id).include?(@coordinator.id) || managed_enterprises.pluck(:id).include?(enterprise.id)
+      managed_enterprises.pluck(:id).include?(@coordinator.id) \
```

Comment:
`\`?

Is that the recommended way of doing this? I thought it would be to have the operator on the first line and then ruby knows to look for the rest in the next line.

Of course there are guidelines. ;-)
https://github.com/rubocop-hq/ruby-style-guide#no-trailing-backslash

Reply:
ah, yes, I saw @kristinalim using this in another PR and I wanted to explore. I just realized that keeping the operator in the first line, as you say, does the trick.